### ISSUE TITLE
Bootstrap tooltip and popover not work when fullscreen

### ISSUE THREAD

**Issue by bagaskarala** [mirari/vue-fullscreen#21]
As stated in title, Bootstrap tooltip and popover not work when fullscreen? How to overcome this?

**Comment by mirari** [comment]
Only child elements in the fullscreen element are visible.
Tooltip popover element is usually under the `body`.
You can set the `body` element to fullscreen, and change the style of the target element when `body` is fullscreen.
e.g.
```
 this.$fullscreen.toggle(document.body)
```
```
<style>
body.fullscreen .target-element {
  position: absolute;
  top: 0;
  right: 0;
  bottom: 0;
  left: 0;
}
</style>
```

**Comment by mirari** [comment]
Now you can try the new feature `teleport` released on version >=2.5.2.
